enforcer/src/main/java/org/wso2/micro/gateway/enforcer/config/EnvVarConfig.java

```diff
@@ -34,10 +34,10 @@
     public static final String XDS_MAX_MSG_SIZE = "XDS_MAX_MSG_SIZE";
 
     // Since the container is running in linux container, path separator is not needed.
-    private static final String DEFAULT_TRUSTED_CA_CERTS_PATH = "/home/wso2/security";
+    private static final String DEFAULT_TRUSTED_CA_CERTS_PATH = "/home/wso2/security/truststore";
     private static final String DEFAULT_ADAPTER_HOST_NAME = "adapter";
     private static final String DEFAULT_ENFORCER_PRIVATE_KEY_PATH = "/home/wso2/security/keystore/mg.key";
-    private static final String DEFAULT_ENFORCER_PUBLIC_CERT_PATH = "/home/wso2/security/truststore/mg.pem";
+    private static final String DEFAULT_ENFORCER_PUBLIC_CERT_PATH = "/home/wso2/security/keystore/mg.pem";
```

Comment:
same change is there in shalki's PR too. there will be a conflict

Reply:
If I revert this, still it would create a conflict. I think best way is to keep it as it is. Since it is the exact change in both PRs it would not create a conflict.